### DISCUSSION TITLE
FullfillmentInbound.GetLabel

### DISCUSSION THREAD

**Discussion by suzieahn1117** [abuzuhri/Amazon-SP-API-CSharp#221] [Q&A] [answered]
Hi.

Thank you for your SP-API nuget packages work. 
I am currently a software developer working in logistics and trying to use your SP-API nuget, but I found something that might mismatch from Amazon SP-API response model.

It is in FulfillmentInbound.GetLabels and GetLabelsAsync().
GetLabels endpoint should return download URL for returned payload.

![image](https://user-images.githubusercontent.com/18728064/176033843-54ebb1a9-cf55-4216-8920-ee643cbc1302.png)
![image](https://user-images.githubusercontent.com/18728064/176033872-75ee8094-0172-475a-a4e2-25ac0686e4cc.png)

But I see in the code returning GetPrepInstructionResult as a response object. 

![image](https://user-images.githubusercontent.com/18728064/176033971-3f867a5a-5dfe-4e2f-bd76-3a1d56852874.png)

I am trying to use it to download shipping or pallet labels from Amazon.
Let me know if I missed out on using the functions.
Thank you!

**Comment by suzieahn1117** [comment, marked as answer]
Never mind.
I updated your packages and see correct response model.

**Reply by abuzuhri** [reply]
Prefect
